lib/msf/core/payload/apk.rb

```diff
@@ -373,6 +373,11 @@ def backdoor_apk(apkfile, raw_payload, signature = true, manifest = true, apk_da
       fix_manifest(tempdir, package, classes['MainService'], classes['MainBroadcastReceiver'])
     end
 
+    filename = "#{tempdir}/original/res/values-v31/colors.xml"
+    text=File.read(filename)
+    content=text.gsub("@android:", "@*android:")
+    File.open(filename,'w') { |file| file << content }
```

Comment:
I don't know that this file is guaranteed to exist so this the read might raise an exception:
```suggestion
    if File.exist?(filename)
      text=File.read(filename)
      content=text.gsub("@android:", "@*android:")
      File.open(filename,'w') { |file| file << content }
    end
```